v1/onthisday.js

```diff
@@ -26,7 +26,23 @@ class Feed extends BaseFeed {
                 [req.params.type]: res.body[req.params.type]
             };
         }
-        return super._hydrateResponse(hyper, req, res);
+        const hydratedResponse = super._hydrateResponse(hyper, req, res);
```

Comment:
It's actually faster to use Objects to determine whether a value should be filtered out:

```js
return super._hydrateResponse(hyper, req, res).then((response) => {
  Object.keys(response.body).forEach((key) => {
    if (!Array.isArray(response.body[key])) {
      return;
    }
    response.body[key].forEach((elem) => {
      if (!elem || !Array.isArray(elem.pages)) {
        return;
      }
      const titles = {};
      elem.pages = elem.pages.filter((item) => {
        if (titles[item.title]) {
          return false;
        }
        titles[item.title] = true;
        return true;
      });
    });
  });
  return response;
});
```

Reply:
@d00rman 
Updated! I kept the declarative naming, but I don't feel super strongly about it if you prefer it more concise.